Documentation/core-repos.md

```diff
@@ -0,0 +1,30 @@
+# .NET Core Repos
+
+There are many repos that make up .NET Core. To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set .NET Core repositories follows.
```

Comment:
"To file an issue, make a PR, or engage on specific topics, it is best to engage with developers working on the right repo. A largely complete set .NET Core repositories follows." is a bit hard to read.
Suggestion:
"Different individuals (or teams) track different repositories. So, you'll get better results if you file an issue, make a PR, or engage on specific topics on the repo specific to that work. The following list shows the main .NET Core repositories:"